documentation/onboard-dpg-in-sdkautomation/.net/README.md

```diff
@@ -0,0 +1,21 @@
+# Add Autorest Configuration of .Net SDK
+
+## Parameter Description
+
+- `<ServiceName>`: The RP name, which is usually same as folder name in swagger.
+- `<NameSpace>`: Python package name.
+- `<Title>`: SDK Client Name
+
+## Autorest Configuration
+Please copy the following configuration into spec PR comment.
+~~~
+# azure-sdk-for-net-track2
```

Comment:
I assume this will match the value inside of the specConfiguration file?

Reply:
yes